src/local_scheduler/local_scheduler_client.cc

```diff
@@ -75,6 +75,7 @@ void local_scheduler_reconstruct_object(LocalSchedulerConnection *conn,
                                         ObjectID object_id) {
   write_message(conn->conn, RECONSTRUCT_OBJECT, sizeof(object_id),
                 (uint8_t *) &object_id);
+  /* TODO(swang): Propagate the error. */
```

Comment:
We should probably put a fatal check around this (and all other `local_scheduler_client` methods). That can be a different PR though.